dvc/scm/git/__init__.py

```diff
@@ -247,11 +247,12 @@ def list_tags(self):
     def list_all_commits(self):
         return [c.hexsha for c in self.repo.iter_commits("--all")]
 
-    def _install_hook(self, name, cmd):
-        command = (
-            '[ "$3" = "0" ]'
```

Comment:
This line was basically breaking other hooks previously, as it only makes sense for post-checkout.